src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx

```diff
@@ -109,61 +108,51 @@ const UserMenu: FC<UserMenuProps> = ({
                 {formatMessage({ id: 'help' })}
               </Button>
             </div>
-            <div className="w-full pb-6 mb-6 border-b border-b-gray-200 sm:pb-5 sm:mb-5">
+            <div className="w-full pb-4 mb-6 border-b border-b-gray-200 sm:pb-3 sm:mb-5">
               <Button mode="quinary" isFullSize onClick={connectWallet}>
                 {formatMessage({ id: 'connectWallet' })}
               </Button>
             </div>
           </>
         )}
-        <div className="w-full pb-6 mb-6 border-b border-b-gray-200 sm:pb-5 sm:mb-5">
-          <TitledContent title={{ id: 'userMenu.optionsTitle' }}>
-            <ul className="text-lg font-semibold sm:font-normal sm:text-md text-left">
-              {userMenuItems.map(({ id, link, icon, name: itemName }) => (
-                <li className="mb-4 last:mb-0" key={id}>
-                  {link ? (
-                    <Link
-                      to={link}
-                      className="flex items-center transition-all duration-normal text-gray-700 hover:text-blue-400"
-                    >
-                      <Icon name={icon} appearance={{ size: iconSize }} />
+        <div className="w-full pb-4 mb-6 border-b border-b-gray-200 sm:pb-3">
+          <TitleLabel text={formatMessage({ id: 'userMenu.optionsTitle' })} />
+          <ul className="text-left">
+            {userMenuItems.map(({ id, link, icon, name: itemName }) => (
+              <li key={id} className="mb-2 last:mb-0 sm:mb-0">
+                {link ? (
+                  <Link to={link} className="navigation-link">
+                    <Icon name={icon} appearance={{ size: 'tiny' }} />
+                    <p className="ml-2">{formatMessage({ id: itemName })}</p>
+                  </Link>
+                ) : (
+                  <button
+                    type="button"
+                    className="navigation-link"
```

Comment:
aria-label is missing

Reply:
aria-label isn't required when inside the button is text, here is a item name